### PR TITLE
modified to handle alternate ports in start/stop scripts

### PR DESCRIPTION
**Pull request by drstrangelooker**
The port 9999 is hardcoded into the scripts in the stop section. I updated this.

### CODE REVIEW

File: startup_scripts/looker_java1.7 [hunk 1]
```diff
@@ -58,14 +60,14 @@ stop() {
         state="running"
         token=`cat .status_server_token`
         request="control/stop?token=${token}"
-        timeout 20 curl -m 10 -ks https://127.0.0.1:9999/${request} > /dev/null 2>&1
+        timeout 20 curl -m 10 -ks https://127.0.0.1:${LLOKERPORT}/${request} > /dev/null 2>&1
```
Comment (npickens) [inline startup_scripts/looker_java1.7]: Typo! And it got copied and pasted twice below too. :-)

**Comment by drstrangelooker** [comment]
D'oh. I tested the regular start but not the Java1.7 version. I fixed the typo. Thanks @npickens.

**Comment by awaage** [comment]
This was also an issue for me. Realized that when I followed instructions found here (https://looker.com/docs/setup-and-management/on-prem-mgmt/upgrade), it was not really stopping looker.

Had to modify to remove https (for http), and change the port.

**Comment by aabrahamian** [comment]
I just noticed this, is this still a desired change?
If so, the merge conflicts will need to be fixed.
If not, we should close it.

Let me know either way and we'll get this taken care of. :)

**Comment by drstrangelooker** [comment]
Still should be done.

**Comment by aabrahamian** [comment]
Cool, let me know when you have fixed the conflicts, retested, and I'll get this merged in.

**Comment by drstrangelooker** [comment]
@aabrahamian just updated.

**Pull request merged**